lib/bosh/workspace/project_deployment.rb

```diff
@@ -63,14 +54,34 @@ def execute_stub(stub_file)
 
     private
 
+    def stubs_dir
+      File.expand_path(File.join(file_dirname, "../stubs"))
+    end
+
+    def stub_file
+      Dir[File.join(stubs_dir, "#{file_basename('.*')}*")].first
```

Comment:
would you mind if stub file will be determined in the following way:

* first look for option `--stub-file`
* look for file in `stubs` folder with the same name as a deployment name
* and only then take the first file